.github/workflows/on_PR_meson.yaml

```diff
@@ -146,12 +146,12 @@ jobs:
           meson compile -C "${{github.workspace}}/build"
           meson test -C "${{github.workspace}}/build"
   FreeBSD:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
What about the

> The runs-on: ubuntu-22.04 must be ubuntu-22.04

What happens when "latest" becomes 24.04 in a few months time?

Reply:
I doubt there will be a difference. vnactions just uses libvirt

Reply:
I guess it still quite some time away,and we'll find out...